### DISCUSSION TITLE
Is there any known way to enable and disable motion sensing on the wyzecam through possibly rtsp or other means locally

### DISCUSSION THREAD

**Discussion by sideup66** [gtxaspec/wz_mini_hacks#719] [General]
General question if anyone is able to answer....is there a way to tell a wyzecam v3 to enable motion sensing over an ssh command? I am currently using IFTTT to push commands to my camera, but at times find ifttt inconsistent at times, and now after troubleshooting, im being locked out by "IFTTT PRO"....which is strange given im just sending a webhook command by tickling a url.... any help on this would be appreciated. Thanks

**Comment by jmbulan** [comment]
by motion sensing do you mean the green box? what animation are you trying to do with iftt?

**Reply by meow4agreencard** [reply]
@sideup66 if you mean the green box, the only way I had found was to disable "Motion tagging" in the official Wyze app.
